exercises/queen-attack/QueenAttackTest.cs

```diff
@@ -1,55 +1,93 @@
-﻿using System;
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
 using Xunit;
+using System;
 
 public class QueenAttackTest
 {
     [Fact]
-    public void Cannot_occupy_same_space()
+    public void Queen_with_a_valid_position()
+    {
+        var actual = QueenAttack.Create(2, 2);
```

Comment:
Perhaps add a comment below this line to indicate that we are testing that no exception is thrown.